payment/locale/fr/LC_MESSAGES/django.po

```diff
@@ -74,18 +88,31 @@ msgstr ""
 "Voici un bref rappel des conditions obligatoires de l'examen surveillé "
 "pour avoir la possibilité d'obtenir le certificat :"
 
-#: /edx/app/edxapp/fun-apps/payment/templates/payment/base.html:0
+#: /edx/app/edxapp/fun-apps/payment/templates/payment/base-abstract.html:0
 msgid "Home"
 msgstr "Accueil"
 
+#: /edx/app/edxapp/fun-apps/payment/templates/payment/list_orders.html:0
+msgid "Nothing has yet been ordered."
+msgstr "Rien n'a encore été commandé."
```

Comment:
Ok, après avoir relu la thread, on va y aller pour la version un peu plus précise tout en étant cohérent avec les autres messages. Mais il faut clore cette discussion. Si ça pose réellement problème on modifiera prochain sprint.

"Vous n'avez pas encore souscrit à d'examen certifiant"/"You have not yet registered to a certified exam".